README.md

```diff
@@ -193,3 +200,134 @@ For more information on running and tearing down k8s environments, follow this [
 
 Because some of our modules pass headers bigger than 4k we had to increase the default value of the proxy buffer size for nginx.
 We also enable the CORS header for the applications that need it through the Ingress Rule.
+
+# Docker for Desktop - Mac
+
+## Prerequisites
+
+| Component   | Recommended version | Getting Started Guide |
+| ------------|:-----------: | ----------------------   |
+| Docker for Desktop | 18.06.1-ce   | https://www.docker.com/products/docker-desktop |
+| Homebrew           | 1.7.6        | https://brew.sh/         |
+| Helm               | 2.8.2        | https://docs.helm.sh/using_helm/#quickstart-guide |
+
+## Setup
+
+After installing Docker for Desktop you must enable Kubernetes and we recommend that you increase the memory and cores used by Docker for Desktop.
+
+- Memory mimimum: 8 GB
+- Core minimum: 4 cores
+
+We also recommend that you use Homebrew to install the Helm prerequisite.
+
+If you have previously deployed the DBP to AWS you will need to change/verify the ```docker-for-desktop``` context is being used.
+
+```bash
+kubectl config current-context                 # Display the current context
+kubectl config use-context docker-for-desktop  # Set the default context
+```
+
+### Helm Tiller
+
+Initialize the Helm Tiller:
+```bash
+helm init
+```
+
+### K8s Cluster Namespace
+
+As mentioned as part of the Anaxes Shipyard guidelines, you should deploy into a separate namespace in the cluster to avoid conflicts (create the namespace only if it does not already exist):
+```bash
+export DESIREDNAMESPACE=example
+kubectl create namespace $DESIREDNAMESPACE
+```
+
+This environment variable will be used in the deployment steps.
+
+## Deployment
+
+### 1. Add the Alfresco Helm repository to helm
+
+```bash
+helm repo add alfresco-incubator https://kubernetes-charts.alfresco.com/incubator
+```
+
+### 2. Add local DNS
+
+Add Local DNS Entry for Host Machine (needed for JWT issuer matching). Be sure to specify an active network interface.  It is not always ```en0``` as illustrated.  You can use the command ```ipconfig -a``` to find an active interface.
+
+```bash
+sudo sh -c 'echo "`ipconfig getifaddr en0`       localhost-k8s" >> /etc/hosts'; cat /etc/hosts
+```
+
+*Note:* If your IP address changes you will need to update the ```/etc/hosts``` entry for localhost-k8s.
+
+### 3. Deploy the DBP
+
+```bash
```

Comment:
Can we add a brief sentence before this on why we're using this long command?

Reply:
Adding description of the command.